packages/devtools_app/lib/src/inspector/layout_explorer/flex/flex.dart

```diff
@@ -101,104 +104,6 @@ class _FlexLayoutExplorerWidgetState extends LayoutExplorerWidgetState<
     });
   }
 
-  Widget _visualizeFlex(BuildContext context) {
```

Comment:
Fyi @scheglov the extract widget refactor almost handled this case perfectly.  It did have trouble adding multiple named parameters with the same name (for example properties) that should have all been one parameter. A reference to this also caused problems and I had to manually add that parameter.